docs/eventing/migration/sinkbinding.md

```diff
@@ -96,3 +102,10 @@ Context Attributes,
 Data,
   {"signal":2,"data":{"brokerage":{"broker":{"id":"BMS-USB-5-JACE","meta":{"protocol":"BACNET","building":"Urban Sciences Building","buildingFloor":"5"}},"id":"Drivers.L6_C3_Electric_Meters.C3_Mechcanical_Plant.points.C3_HP_Current_L1","meta":{}},"entity":{"name":"Urban Sciences Building: Floor 5","meta":{"building":"Urban Sciences Building","buildingFloor":"5"}},"feed":{"metric":"C3 HP Current L1","meta":{}},"timeseries":{"unit":"no units","value":{"time":"2020-03-05T13:45:51.468Z","timeAccuracy":8.754,"data":0.47110211849212646,"type":"Real"}}},"recipients":0}
 ```
+
+## When to use SinkBinding?
+
+From the above options, `ContainerSource` is arguably the simplest one. 
+However, the true power of `SinkBinding` comes from the fact that it can work not just with `Deployments` 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
However, the true power of `SinkBinding` comes from the fact that it can work not just with `Deployments`
```